projects/distal_input/NAB/config/profiles.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Better would be to import this from NAB, and use this directory for anything explicitly different from NAB. Repeated code and documentation can easily lead to inconsistencies and issues.